boot.php

```diff
@@ -686,11 +687,12 @@ function __construct() {
 			$this->hostname = $hostname;
 		}
 
-		if (is_array($_SERVER["argv"]) && $_SERVER["argc"]>1 && substr(end($_SERVER["argv"]), 0, 4)=="http" ) {
-			$this->set_baseurl(array_pop($_SERVER["argv"]) );
+		if (is_array($_SERVER["argv"]) && $_SERVER["argc"] > 1 && substr(end($_SERVER["argv"]), 0, 4) == "http" ) {
+			$this->set_baseurl(array_pop($_SERVER["argv"]));
 			$_SERVER["argc"] --;
 		}
 
+		/// @TODO no longer needed?
 		#set_include_path("include/$this->hostname" . PATH_SEPARATOR . get_include_path());
 
 		if ((x($_SERVER,'QUERY_STRING')) && substr($_SERVER['QUERY_STRING'],0,9) === "pagename=") {
```

Comment:
If I search for `$_SERVER,'`, I can find 10 occurrences scattered over several files. As usual I fix it here only and add it to my todos.txt file.